node/rust-client/src/lib.rs

```diff
@@ -257,7 +258,9 @@ pub fn create_leaked_string(message: String) -> [u32; 2] {
 
 #[napi(ts_return_type = "[number, number]")]
 pub fn create_leaked_string_vec(message: Vec<String>) -> [u32; 2] {
-    let pointer = Box::leak(Box::new(message)) as *mut Vec<String>;
+    // Convert the string vec -> Bytes vector
```

Comment:
I think argument for this function should be `byte[][]`

Reply:
Thanks, I was thinking of fixing this in the next PR. But it makes no difference :)

Fixed